pupil-spa/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
do we need editorconfig? isn't standardJS covering most of this when used on the IDE(webstorm etc)